src/Symfony/Component/Mailer/Bridge/Sendgrid/Transport/SendgridSmtpTransport.php

```diff
@@ -27,7 +27,13 @@ class SendgridSmtpTransport extends EsmtpTransport
 {
     public function __construct(#[\SensitiveParameter] string $key, ?EventDispatcherInterface $dispatcher = null, ?LoggerInterface $logger = null, private ?string $region = null)
     {
-        parent::__construct(null !== $region ? \sprintf('smtp.%s.sendgrid.net', $region) : 'smtp.sendgrid.net', 465, true, $dispatcher, $logger);
+        $host = 'smtp.sendgrid.net';
+
+        if (null !== $region && 'global' !== strtolower($region)) {
```

Comment:
```suggestion
        if (null !== $region && 'global' !== $region) {
```

I don't think we need to make it case insensitive.